### DISCUSSION TITLE
What does `“Code”` mean in the CLA?

### DISCUSSION THREAD

**Discussion by schollz** [microsoft/winget-pkgs#155182] [General] [answered]
Hi,

I find myself encouraged by users of one of my pieces of software to submit it to winget. It seems the PR to winget requires agreeing to a CLA. Before agreeing to any contract, I try to understand what I am agreeing to, but in this case I find that I cannot understand. I do not understand this definition provided in the CLA:

> “Code” means the computer software code, whether in human-readable or machine-executable form,
that is delivered by You to Microsoft under this Agreement.

Does this CLA `“Code”` mean the code associated with the PR to winget (in my case that consist entirely of [autogenerated YAML files](https://github.com/microsoft/winget-pkgs/pull/155177/files))?

Or...

Does this CLA `“Code”` mean the code associated with the code reference within the YAML files (in my case that would be the code at https://github.com/schollz/croc)?

**Comment by stephengillie** [comment, marked as answer]
I'm not a lawyer, but my understanding is that "Code" means the YAML files, as they are your (or your employer's) creation. And so you have to give permission to the package manager project to use your creation and host it in the repo.
